src/lib/holocene/accordion.svelte

```diff
@@ -24,7 +31,7 @@
         </h2>
         <Icon
           name={open ? 'caretUp' : 'caretDown'}
-          stroke="currentcolor"
+          stroke={disabled ? colors.gray[500] : 'currentcolor'}
```

Comment:
Nice I like this, usually I just look up the hex value in our config and hard code it 😂

Reply:
That way is probably more performant 😅